content/article/promoting-perl-community-articles.md

```diff
@@ -42,7 +42,7 @@ Last week I added the "Community Articles" toolbar to this website. It's a JavaS
 
 The list of links is updated hourly, served over HTTPS and hosted on GitHub pages. The widget is clever enough to skip links to articles from the host domain too (it never shows links to PerlTricks.com on this site).
 
-If you'd rather not add that sidebar, consider adding a [widget](https://www.reddit.com/r/perl/widget) for `/r/perl`. The Reddit widget respects Do Not Track.
+If you'd rather not add that sidebar, consider adding a [widget](http://web.archive.org/web/20170504023532/https://www.reddit.com/r/perl/widget) for `/r/perl`. The Reddit widget respects Do Not Track.
```

Comment:
Let's not fix the reddit thing here because the historical resource doesn't matter. Remove the link and add a note like *Editor's note: Reddit does not support widgets anymore*

Reply:
Here are the requested changes:

`The list of links is updated hourly, served over HTTPS and hosted on GitHub pages. The widget is clever enough to skip links to articles from the host domain too (it never shows links to PerlTricks.com on this site).

If you'd rather not add that sidebar, consider adding a  **Editor's Note: Reddit does not support widgets anymore** for `/r/perl`. The Reddit widget respects Do Not Track.

Finally, participate on  **See Editor's Note** and Twitter! Link to Perl resources and content you like; upvote or retweet Perl-related tweets. If you have something to say, reply to tweets or add yours comments to the subreddit links. Use the `#perl` hashtag.`

Pls let me know if this is not the correct way to update this code.  Do I start a new pull request and attached the updated files?   Pls let me know.

Reply:
[promoting-perl-community-articles.zip](https://github.com/tpf/perldotcom/files/5901351/promoting-perl-community-articles.zip)

Adding zip file

Reply:
Huh, this file isn't even the repo right now.

Reply:
Changed to much here. Make it 

> If you'd rather not add that sidebar, consider adding a widget for `/r/perl` (Editor's note: Reddit doesn't do this anymore).